src/recordlinker/linking/mpi_service.py

```diff
@@ -57,6 +57,21 @@ def get_block_data(
     return session.execute(expr).scalars().all()
 
 
+def fetch_person_records(
```

Comment:
We should write a test for this function just to make sure its working as expected